doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -48,6 +48,39 @@ Registering Fast Pair Provider
 ==============================
 
 See the official `Fast Pair Model Registration`_ documentation for information how to register the device and obtain the Model ID and Anti-Spoofing Public/Private Key pair.
+Alternatively, you can use debug Model ID and Anti-Spoofing Public/Private Key pair obtained by Nordic for development purpose.
+Following debug devices have been registered:
+
+1. NCS Fast Pair demo - general purpose Fast Pair provider. By default it's used by the :ref:`peripheral_fast_pair` sample. It has following information:
+
+   * Device name: NCS Fast Pair demo
+   * Model ID: 0x2A410B
```

Comment:
Maybe it would be nice to enclose also model IDs inside backticks` ``<model_id>`` ` as suggested already for Anti-spoofing key?